src/utils/miscellaneous.ts

```diff
@@ -51,3 +52,9 @@ export function getImageUrl(tokenAddress?: string): string | undefined {
 }
 
 export const isOrderActive = (order: AuctionElement, now: Date): boolean => batchIdToDate(order.validUntil) >= now
+
+export function isOrderFilled(order: AuctionElement): boolean {
+  // consider an oder filled when 99% or more is matched
```

Comment:
why 99%?

Can we make it a bit bigger? Like with that. If we sell 1M DAI we can have 10K DAI still pending.
Maybe I would sth like `99.99999999999%` :) 

Or an alternative is sth in the direction of: `99.99%` and remaining amount `< 0.01` whatever is more restrictive

Reply:
> why 99%?
So I got feedback on what a better value would be :)

Will do 0.01%

Reply:
Updated. Oh, just noticed you also mentioned a fixed value. Updating...

Reply:
Actually, that makes it much more complicated. I need the token decimals to calculate what's the amount left, and that's not part of the order object.
Can we stick to just the 0.01% for now?

Reply:
Although I would make it even smaller `0.0001%`
`0.0001%` of `100 Ether` --> 0.0001 ETH --> `0.026612 $` at current price

Because the ` 0.01%` would be:
`0.01%` of `100 Ether` --> 0.01 ETH --> `2.6612 $` at current price

Reply:
I could argue that relatively speaking, $ 2.6612 out of $ 26,612.00 is negligible, but I won't :D

Instead I'll lower down from 0.01% to 0.0001%